f3.go

```diff
@@ -187,11 +188,18 @@ func (m *F3) Start(startCtx context.Context) (_err error) {
 	}
 	{
 		var maybeNetworkName gpbft.NetworkName
+		var bootstrapEpoch int64
+		var finality int64
+		var initialPowerTable *cid.Cid
 		if m := m.manifest.Load(); m != nil {
 			maybeNetworkName = m.NetworkName
+			bootstrapEpoch = m.BootstrapEpoch
+			finality = m.EC.Finality
+			initialPowerTable = &m.InitialPowerTable
 		}
-		log.Infow("F3 is starting", "initialDelay", initialDelay,
-			"hasPendingManifest", hasPendingManifest, "NetworkName", maybeNetworkName)
+		log.Infow("F3 is starting", "initialDelay", initialDelay, "hasPendingManifest", hasPendingManifest,
+			"NetworkName", maybeNetworkName, "bootstrapEpoch", bootstrapEpoch, "finality", finality,
+			"initialPowerTable", initialPowerTable)
 	}
```

Comment:
At this point I'd have two logs: one inside the condition and one outside.

Reply:
Good idea! Fixed.